cypress/component/DarCollectionTable/researcher_actions.spec.js

```diff
@@ -4,167 +4,140 @@
 import { cloneDeep } from 'lodash/fp';
 import ResearcherActions from '../../../src/components/dar_collection_table/ResearcherActions';
 
-let propCopy;
-const collectionId = 1;
-const collectionSkeleton = {
-  darCollectionId: collectionId,
-  dars: undefined,
-};
-
-const canceledDars = {
-  1: {
-    data: {
-      status: 'Canceled'
-    }
-  },
-  2: {
-    data: {
-      status: 'Canceled'
-    }
-  }
-};
-
-const darsWithElections = {
-  1: {
-    data: {},
-    elections: {
-      1: { status: 'Open' }
-    }
-  },
-  2: {
-    data: {},
-    elections: {
-      2: { status: 'Closed' }
-    }
-  }
-};
-
-const darsWithNoElections = {
-  1: {
-    data: {},
-    elections: {}
-  },
-  2: {
-    data: {},
-    elections: {}
-  }
-};
-
-const mixedCancelElectionDars = {
-  1: {
-    data: {
-      status: 'Canceled'
-    },
-    elections: {
-      1: { status: 'Closed'}
-    }
-  },
-  2: {
-    data: {
-      status: 'Canceled'
-    }
-  }
-};
 
-const props = {
-  collection: collectionSkeleton,
+const draftDarColl = {
+  "darCollectionId": null,
+  "referenceIds": [
+    '0a4jn-g838d-bsdg8-6s7fs7',
+  ],
+  "darCode": "DRAFT-023",
+  "name": null,
+  "submissionDate": "2022-07-26",
+  "researcherName": null,
+  "institutionName": null,
+  "status": "Draft",
+  "actions": [
+    "Resume",
+  ],
+  "hasVoted": false,
+  "datasetCount": 10
+}
+
+const darColl = {
+  "darCollectionId": 2345,
+  "referenceIds": [
+    '4a3fd-g77fd-2f345-4h2g31',
+    '0a4jn-g838d-bsdg8-6s7fs7',
+  ],
+  "darCode": "DAR-9583",
+  "name": "Example DAR 1",
+  "submissionDate": "2022-07-26",
+  "researcherName": "John Doe",
```

Comment:
Codacy has a fix for the issue: [Strings must use singlequote.](https://app.codacy.com/gh/DataBiosphere/duos-ui/pullRequest?prid=9866666)
```suggestion
  'researcherName': "John Doe",
```